BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Networking/GiniBank+Networking.swift

```diff
@@ -70,12 +70,14 @@ extension GiniBank {
                                      resultsDelegate: GiniCaptureResultsDelegate,
                                      documentMetadata: Document.Metadata? = nil,
                                      trackingDelegate: GiniCaptureTrackingDelegate? = nil,
-                                     networkingService: GiniCaptureNetworkService) -> UIViewController {
+                                     networkingService: GiniCaptureNetworkService,
+                                     configurationService: ConfigurationServiceProtocol? = nil) -> UIViewController {
         let screenCoordinator = GiniBankNetworkingScreenApiCoordinator(resultsDelegate: resultsDelegate,
                                                                        configuration: configuration,
                                                                        documentMetadata: documentMetadata,
                                                                        trackingDelegate: trackingDelegate,
-                                                                       captureNetworkService: networkingService)
+                                                                       captureNetworkService: networkingService,
+                                                                       configurationService: configurationService)
         AnalyticsManager.firstSDKOpen = true
         return screenCoordinator.start(withDocuments: importedDocuments)
```

Comment:
[suggestion]: we also need here to call `return screenCoordinator.startSDK(withDocuments: importedDocuments)`